common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2111,8 +2111,35 @@
   },
   "INVALID_EMPTY_LOCATION" : {
     "message" : [
-      "The location name cannot be empty string, but `<location>` was given."
+      "The location cannot be empty string or null."
```

Comment:
How about: `The location `<location>` cannot be empty string or null for`. And in sub classes we just put message as `table.`, `partition.`, etc.

Reply:
Do we still need subclasses?

Reply:
good question. Seems we don't need to change this error condition at all :)